src/DragDropContainer.jsx

```diff
@@ -170,9 +170,10 @@ class DragDropContainer extends React.Component {
 	drag = (x, y) => {
 		const { left, top } = this.state;
 		const { xOnly, yOnly, dragData, onDrag } = this.props;
+		const undraggableZone = 3;
```

Comment:
Is 3 enough?))) Please, ask Liza to test this task)

Reply:
it is less then 3) but for my mouse is enough)

Reply:
If it's ok on Liza's PC, then we are safe)))
Last time Anton also said that it's ok for him))))